lib/catalog/rbac/access.rb

```diff
@@ -21,9 +23,24 @@ def destroy_access_check
         resource_check('delete')
       end
 
+      def admin_check
+        return true unless rbac_enabled?
+
+        return catalog_admin?
+      end
+
+      def group_check
```

Comment:
@eclarizio Shouldn't this get removed now?

Reply:
Yes you're right, this is unused and useless now.